Game/RainMeadow.CreatureHooks.cs

```diff
@@ -202,5 +206,36 @@ private void TentaclePlantAI_Update(On.TentaclePlantAI.orig_Update orig, Tentacl
             orig(self);
             if (!self.creature.IsLocal()) self.mostInterestingItem = mostInterestingItem;
         }
+
+        // HACK: doesn't play sounds, we should IL hook to disable just the eggs
+        private void EggBug_DropEggs(On.EggBug.orig_DropEggs orig, EggBug self)
+        {
+            if (!self.IsLocal())
```

Comment:
Because we're checking OPOs won't we need a game mode check?

Reply:
if it's a remote creature then remote will be in charge of simulating it, gamemode doesn't change this

Reply:
Is it worth wrapping it in a game mode check (or at least it online mode is not null) so we don't run operations period in single player story?

Reply:
IsLocal returns true if offline